### ISSUE TITLE
Add support Solana blockchain

### ISSUE THREAD

**Issue by gyengus** [codyborn/tokenCast#22]
Solana is growying rapidly. There are more NFT started on Solana blockchain. I think, it will be useful, if this application supports it.

**Comment by zypherion** [comment]
Agreed. An eth project I follow is releasing new batches of nfts on SOL. Would be nice to be able to show a mix of SOL and ETH nfts on one screen.
